packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.1.16 2020-02-12
+* Add zoomable timeline grid and timestamps to flame chart. (#1624)
```

Comment:
Can we normalize punctuation of commit messages? Perhaps check that a commit message ends in a period and if not, add one. Or the opposite - check if a commit message ends in a period and if so, remove it. 

Option 1 sounds easier to me.

Reply:
Option 2 ended up being simpler to write (easy to check for the existence of a ". (#" pattern without a regex).  Done.